IS7024_XML_Group_Project/Pages/Index.cshtml.cs

```diff
@@ -26,29 +26,28 @@ public void OnGet()
         {
            double temp = 0;
 
-
-
-            //pull in JSON stream for candy
-            using (var webClient = new WebClient())
-            {
-                string jsonString = webClient.DownloadString("https://raw.githubusercontent.com/UC-Classes/IS7024_XML_Group_Project/master/candyfile.txt");
-                var candy = Candy.FromJson(jsonString);
-                ViewData["Candy"] = candy;
+           //pull in JSON stream for candy
+           string jsonString = DownloadData("https://raw.githubusercontent.com/UC-Classes/IS7024_XML_Group_Project/master/candyfile.txt");
+           var candydata = Candy.FromJson(jsonString);
+           ViewData["Candy"] = candydata;
             
 
             // get our weather data and key
             string weatherAPIKey = System.IO.File.ReadAllText("WeatherAPIKey.txt");
-            string weatherData = webClient.DownloadString("https://api.weatherbit.io/v2.0/current?&city=Cincinnati&country=USA&key=" + weatherAPIKey);
+            string weatherData = DownloadData("https://api.weatherbit.io/v2.0/current?&city=Cincinnati&country=USA&key=" + weatherAPIKey);
 
             // parse to objects
             QuickTypeWeather.Weather weather = QuickTypeWeather.Weather.FromJson(weatherData);
             QuickTypeWeather.Datum[] allWeatherData = weather.Data;
 
             //check temperature from weather data
-                foreach (QuickTypeWeather.Datum datum in allWeatherData)
-                {
-                    temp = datum.Temp;
-                }
+            //   foreach (QuickTypeWeather.Datum datum in allWeatherData)
```

Comment:
Don't commit commented out code.   Just delete it.  We can always go back in history in version control to see what used to be there.